### PR TITLE
OSDOCS-2506: architecture overview

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.6 version
- [Jira](https://issues.redhat.com/browse/OSDOCS-2506)
- [Preview](https://deploy-preview-40014--osdocs.netlify.app/openshift-enterprise/latest/architecture/)

### CODE REVIEW

**Comment by wsun1** [comment]
QE reviewed , look good to me.

File: architecture/index.adoc [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+toc::[]
+
+{product-title} is a cloud-based Kubernetes container platform.
+The foundation of {product-title} is based on Kubernetes and therefore shares the same technology.
+To learn more about {product-title} and Kubernetes, see xref:../architecture/architecture.adoc#architecture[product architecture].
+
+[id="about-installation-and-updates"]
+== About installation and updates
+
+As a cluster administrator, you can use {product-title} xref:../architecture/architecture-installation.adoc#architecture-installation[installation program] to install and deploy a cluster on:
```
Comment (bergerhoffer) [inline architecture/index.adoc:15]: you can use -> you can use the

File: architecture/index.adoc [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+** Installer-provisioned infrastructure clusters
+** User-provisioned infrastructure clusters
+
+[id="about-control-planes"]
+== About the control plane
+
+The  xref:../architecture/control-plane.adoc#control-plane[control plane] manages the worker nodes and the pods in your cluster. You can configure nodes with the use of machine config pools (MCPs).
+MCPs are groups of machines such as, control plane components or user workloads, hat are based on the resources that they handle.
```
Comment (bergerhoffer) [inline architecture/index.adoc:24]: * machines such as, -> machines, such as
* hat are -> that are

File: architecture/index.adoc [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+{product-title} assigns hosts different roles to hosts. These  roles define the function of a machine in a cluster.
```
Comment (bergerhoffer) [inline architecture/index.adoc:25]: assigns hosts different roles -> assigns different roles

File: architecture/index.adoc [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+The cluster contains definitions for the standard control plane and worker role types.
+
+You can use Operators to package,deploy, and manage services on the control plane.
```
Comment (bergerhoffer) [inline architecture/index.adoc:28]: Needs a space in "package,deploy"

File: architecture/index.adoc [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+Operators are important components in {product-title} because they provide the following services:
+
+* Perform health checks
+* Provide ways to watch the applications
```
Comment (bergerhoffer) [inline architecture/index.adoc:32]: Remove "the" to be consistent with the 4.9 version

File: architecture/index.adoc [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+* Manage over-the-air updates
+* Ensures applications stay in the specified state
```
Comment (bergerhoffer) [inline architecture/index.adoc:34]: Ensures -> Ensure (to be consistent with 4.9 version)

**Review by bergerhoffer — COMMENTED** [review]
@Amrita42 When comparing to the 4.9 PR (#40012), found a few differences, if you wouldn't mind resolving those prior to merging.

**Comment by bergerhoffer** [comment]
LGTM now, merging!

**Pull request merged**